_posts/2018-01-27-Honduran-Baleada.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Honduran Baleada"
+date: 2018-01-27
+author: me
+permalink: ./some-food/
```

Comment:
Choose a unique link that describes your food. Maybe ``./honduran-baleada/`` would work.